examples/logging-opentelemetry/keystone.ts

```diff
@@ -28,9 +28,12 @@ export default config<TypeInfo>({
 
             return {
               async willSendResponse({ operation, request }) {
-                span.setAttribute('type', operation?.operation || 'unknown')
-                span.setAttribute('name', request.operationName || 'unknown')
-                span.setAttribute('hash', request.query ? sha256(request.query) : 'empty')
+                span.setAttribute('gql.query.type', operation?.operation || 'unknown')
+                span.setAttribute('gql.query.name', request.operationName || 'unknown')
+                span.setAttribute(
+                  'qql.query.sha256',
+                  request.query ? sha256(request.query) : 'empty'
```

Comment:
This should probably be `graphql.query.*` if we want to be similar to what Apollo is using internally, but, for now this is OK